src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopyList = [];
+
+  const oneAction = (task) => {
```

Comment:
Also, you don't need to create a new function, you can just use for of for actions

Reply:
Thanks, solved, but works only inside the loop, otherway array has the same object that times that we have rules in actions